app/serializers/user_serializer.rb

```diff
@@ -12,4 +12,11 @@ class UserSerializer
   attributes :services, :account, :profile, :va_profile, :veteran_status,
              :in_progress_forms, :prefills_available, :vet360_contact_information,
              :session, :onboarding
+
+  def account
```

Comment:
We shouldn't expand usage of the `user.account` object, it's a deprecated object that will eventually be removed